doctr/datasets/vocabs.py

```diff
@@ -31,3 +31,6 @@
 VOCABS['german'] = VOCABS['english'] + 'äöüßÄÖÜẞ'
 VOCABS['arabic'] = (VOCABS['digits'] + VOCABS['hindi_digits'] + VOCABS['arabic_letters'] + VOCABS['persian_letters'] +
                     VOCABS['arabic_diacritics'] + VOCABS['arabic_punctuation'] + VOCABS['punctuation'])
+VOCABS['vietnamese'] = (VOCABS['english'] +
+'áàảạãăắằẳẵặâấầẩẫậéèẻẽẹêếềểễệóòỏõọôốồổộỗơớờởợỡúùủũụưứừửữựiíìỉĩịýỳỷỹỵ' +
```

Comment:
@calibretaliation 
Can we indent line 34 and 35 please ? :sweat_smile:

Reply:
Thanks you soooo much <3